### ISSUE TITLE
[CAN'T SIGN IN TO KAKAOPAGE KOREA]

### ISSUE THREAD

**Issue by jsobari** [manga-download/hakuneko#3252]
after i sign in successfully, it logs me out again. i can't open any chapters from the web and i can't download any locked chapters.

**Comment by 27136773** [comment]
If you login within the browser and then open the website in a new tab, do you require to login again or is the new tab also logged in?

----

It is possible that the website is a SPA, that keeps the login only within the application scope and this would cause some trouble for HakuNeko using manual website interaction to login...

**Comment by Ever4engel** [comment]
ralated to #3208

**Comment by Ever4engel** [comment]
yes i have an account, can't login, even if i do like as you described ronny

**Comment by ap-sensei** [comment]
I am having the same problem too.
If I login in the browser, and open any tab it stays logged-in. 

As you know, its login page opens in a New Window and uses OAuth protocol.
Also, after the successful login, the main window is not refreshed, only the account menu appears (I guess it's AJAX)

It appears that in Haku-Neko the success response of login is not sent back to the main window.

**Comment by 27136773** [comment]
If they use oAuth instead of the classic session cookie, then this would explain why HakuNeko does not work. It would require to send the auth token Header for each request which HakuNeko is not doing right now.

**Comment by ap-sensei** [comment]
So, is there a solution for it that we can look forward to?
Because this website is really useful
Just in case others don't know, to download each chapter you should login with different accounts. It only allows one chapter per 3 days!

**Comment by 27136773** [comment]
I don't think this will be fixed anytime soon

**Comment by ap-sensei** [comment]
Yeah, I assumed too.
But anyway, I know PHP (above average, not an expert) and also some jQuery AJAX.
I don't know if it will help in any way, but tell me. I will do what I can.

**Comment by MikeZeDev** [comment]
use haruneko.
